cs/cli/vw_label.h

```diff
@@ -70,9 +71,14 @@ public ref class ContextualBanditLabel sealed : ILabel
 
     void set(float value)
     { if (value < 0 || value >1)
-        throw gcnew ArgumentOutOfRangeException("invalid probability: " + value);
-
-      m_probability = value;
+      {
+        if (value > 1 && value - 1 < tollerance)
```

Comment:
Consider extracting out to a float equals function that takes into account precision tolerance

Reply:
This line is slightly different then an equal function since the tolerance is only on the right side. I.e., it's not abs(value - 1) < tolerance.
Also, it is only used here, so I think abstracting it out it's a little bit overkill for now. Maybe for the future if we are using this over and over in many places. What do you think?

Reply:
That makes sense! Yeah let’s leave this